Fast and Accurate GPU Quantization for Transformers.md

```diff
@@ -1,25 +1,10 @@
-## Introduction
-As transformer models increase in size, the computational cost of running inference also grows. Many organisations now face the challenge of deploying state-of-the-art models in a cost-effective way.
+As Transformer models increase in size, the computational cost of running inference also grows. Many organisations now face the challenge of deploying state-of-the-art models in a cost-effective way.
 
-This has led to a surge in interest for optimizing transformer inference. There are a range of techniques available [1], including:
+One popular technique for this is **quantization** - by reducing the precision of network parameters and activations, quantization aims to **increase throughput** and **decrease memory footprint**. Whilst reducing peak memory by quantizing from FP16 to INT8 is pretty much guaranteed, achieving performance improvements can be trickier. This is particularly true when running INT8 operations on GPU.
 
-* Sparsity
-* Pruning
-* Conditional Computation (e.g., Mixture of Experts)
-* Knowledge Distillation
-* Quantization
+The main challenge stems from the overheads associated with the quantization/dequantization operations - which can mask the performance gains from the blazingly fast INT8 Tensor Core matrix multiplies. The second part of this blog explores the nuances around achieving peak performance on GPU, and can be found here.
```

Comment:
It seems strange to me that we introduce the second half of the blog before the first half. Is there a way to swap them around?

Reply:
Yeah this is a bit weird, it came from an earlier version where I had split it into two separate blogs and made more sense there. I basically wanted to make it obvious that they could be read almost independently for people that understand quantization already. Will try and rework it

Reply:
Lets review this on main branch